src/explore-education-statistics-common/src/validation/__tests__/createErrorHelper.test.ts

```diff
@@ -269,9 +271,11 @@ describe('createErrorHelper', () => {
             { content: 'Error two' },
             { content: 'Error three' },
           ],
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Remove the anys and try this: `createErrorHelper<{ subjects: [undefined, { content: string }, { content: string }]}>`